examples/dreambooth/README.md

```diff
@@ -376,18 +376,15 @@ After training, LoRA weights can be loaded very easily into the original pipelin
 load the original pipeline:
 
 ```python
-from diffusers import DiffusionPipeline, DPMSolverMultistepScheduler
-import torch
-
-pipe = DiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16)
-pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config)
-pipe.to("cuda")
+import os
+from diffusers import StableDiffusionPipeline
+pipe = StableDiffusionPipeline.from_pretrained(os.environ["MODEL_NAME"]).to("cuda")
```

Comment:
Let's not change these things please.

Reply:
I can remove `os`, but I think it's better to switch it to StableDiffusionPipeline
It's a sd pipeline after all. And you don't need this wizardry with DPMSolverMultistepScheduler

Reply:
It's okay to use `DiffusionPipeline` here since it's more generic. I am not sure how that is related to any wizardry.